### ISSUE TITLE
Simulated Optical Flow using real clouds

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/satflow#88]
As mention in #85 one pre-training idea is to create a flow dataset to pre-train on using clouds. We would need simulated flow, and would want to have realistic clouds in all spectral channels. Easiest way to do that, I think, would be to take real clouds and then crop/paste them over the base ground image and move them to create it. So, to do this, there needs to be 

- [ ] Get clean base image of ground for all timesteps
- [ ] Get set of real clouds that we can use for copy/paste, has to be available for all timesteps so its still the correct pixel values as the day goes on
- [ ] Create augmentation pipeline to generate images on fly and optical flow

**Comment by jacobbieker** [comment]
To help, should also add losses for optical flow, like end point error (e2e), one example is here: https://github.com/NVIDIA/flownet2-pytorch/blob/master/losses.py

**Comment by jacobbieker** [comment]
https://arxiv.org/pdf/1511.05440.pdf%5D Also goes into only using loss on parts of the image above a certain optical flow threshold

**Comment by JackKelly** [comment]
Some very hand-wavy ideas about subtracting 'cloudy pixels' from 'background' here: https://github.com/openclimatefix/predict_pv_yield/issues/17

**Comment by JackKelly** [comment]
I love these ideas!

But, before diving into data augmentation, is there strong reason to believe the models are overfitting?   (although I remember the Perceiver paper emphasising that the Perceiver likes to overfit more than most deep learning models!)
